### PR TITLE
explain how to configure containerd and docker to use runv

### PR DESCRIPTION
**Pull request by bergwolf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ root         4  0.0  1.6  15572  2032 pts/0    R+   05:57   0:00 ps aux
 
 `runv` is a runtime implementation of [OCI runtime](https://github.com/opencontainers/runtime-spec) and its command line is almost compatible with the runc-0.1.1(keeping updated with the newest released runc). It is still under development and uncompleted, such as container tty is not working currently.
 
-Note, runv project also provides [other smoothly way](https://github.com/hyperhq/runv/tree/master/containerd) to integrate with docker.
+Note, runv project also provides other [smooth way](docs/configure-runv-with-containerd-docker.md) to integrate with later versions of docker and containerd.
 
 Example(requires docker-1.11 who talks runc-0.1.1 command line):
```
Comment (laijs) [inline README.md]: does the following example need to be updated?

File: docs/configure-runv-with-containerd-docker.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Configure containerd and docker to run with runv
+
+Because ![runv](../README.md) is an [OCI](https://www.opencontainers.org/) compatible runtime, it can work smoothly with other OCI-compatible management tools, namingly [containerd](https://containerd.io/) and [docker](https://www.docker.com/community-edition), by replacing the default runtime [runc](https://github.com/opencontainers/runc).
+
+## Work with containerd
```
Comment (laijs) [inline docs/configure-runv-with-containerd-docker.md]: we need two:
Work with (the newest) containerd
Work with the docker-containerd of the newest docker

File: docs/configure-runv-with-containerd-docker.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+First, install docker CE as illustrated in the [docker CE download page](https://www.docker.com/community-edition#/download). As of writing, docker CE version `17.06.1-ce` is used.
+
+Stop docker service (with e.g., `systemctl stop docker`) and then create docker's default configuration file `/etc/docker/daemon.json` with following contents, or add following contents to it if it exits already.
+```
+{
+  "default-runtime": "runv",
+  "runtimes": {
+    "runv": {
+      "path": "runv"
```
Comment (laijs) [inline docs/configure-runv-with-containerd-docker.md]: Please add some comment about hyperstart.
Add some information about where the kernel/initrd should be saved.

**Comment by bergwolf** [comment]
@laijs pr updated.

**Comment by laijs** [comment]
LGTM

**Comment by laijs** [comment]
LGTM

**Pull request merged**